From: Andrew Morton <akpm@linux-foundation.org>
Subject: notifiers-add-tracepoints-to-the-notifiers-infrastructure-checkpatch-fixes
Date: Tue Mar 14 01:51:03 PM PDT 2023

ERROR: "(foo*)" should be "(foo *)"
#117: FILE: kernel/notifier.c:43:
+	trace_notifiers_register((void*)n->notifier_call);

ERROR: "(foo*)" should be "(foo *)"
#125: FILE: kernel/notifier.c:53:
+			trace_notifiers_unregister((void*)n->notifier_call);

ERROR: "(foo*)" should be "(foo *)"
#133: FILE: kernel/notifier.c:92:
+		trace_notifiers_run((void*)nb->notifier_call);

total: 3 errors, 0 warnings, 99 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/notifiers-add-tracepoints-to-the-notifiers-infrastructure.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: "Guilherme G. Piccoli" <gpiccoli@igalia.com>
Cc: Guilherme G. Piccoli <kernel@gpiccoli.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/kernel/notifier.c~notifiers-add-tracepoints-to-the-notifiers-infrastructure-checkpatch-fixes
+++ a/kernel/notifier.c
@@ -40,7 +40,7 @@ static int notifier_chain_register(struc
 	}
 	n->next = *nl;
 	rcu_assign_pointer(*nl, n);
-	trace_notifier_register((void*)n->notifier_call);
+	trace_notifier_register((void *)n->notifier_call);
 	return 0;
 }
 
@@ -50,7 +50,7 @@ static int notifier_chain_unregister(str
 	while ((*nl) != NULL) {
 		if ((*nl) == n) {
 			rcu_assign_pointer(*nl, n->next);
-			trace_notifier_unregister((void*)n->notifier_call);
+			trace_notifier_unregister((void *)n->notifier_call);
 			return 0;
 		}
 		nl = &((*nl)->next);
@@ -89,7 +89,7 @@ static int notifier_call_chain(struct no
 			continue;
 		}
 #endif
-		trace_notifier_run((void*)nb->notifier_call);
+		trace_notifier_run((void *)nb->notifier_call);
 		ret = nb->notifier_call(nb, val, v);
 
 		if (nr_calls)
_
